packages/starlight/components/Search.astro

```diff
@@ -53,6 +53,8 @@ const pagefindTranslations = {
 </site-search>
 
 <script>
+	import { formatPathNoBase } from '../utils/format-path';
```

Comment:
I know it might require some code duplication but I’m hesitant about importing this utility into client-side code. Can we move this into a minimal version of the code just for this use case inside the `<script>` tag?

Reply:
Sure!

Reply:
Yes. Basically to isolate the client-side script from modules that are used elsewhere and might import all kinds of things unintentionally at some point.